.buildkite/nightly-benchmarks/scripts/summary-nightly-results.py

```diff
@@ -17,10 +17,24 @@
     "request_throughput": "Tput (req/s)",
     "mean_ttft_ms": "Mean TTFT (ms)",
     "std_ttft_ms": "Std TTFT (ms)",
+    "p99_ttft_ms": "P99 TTFT (ms)",
+    "median_ttft_ms": "Median TTFT (ms)",
     "mean_itl_ms": "Mean ITL (ms)",
     "std_itl_ms": "Std ITL (ms)",
+    "median_itl_ms": "Median ITL (ms)",
+    "p99_itl_ms": "P99 ITL (ms)",
+    "mean_tpot_ms": "Mean TPOT (ms)",
+    "std_tpot_ms": "Std TPOT (ms)",
+    "median_tpot_ms": "Median TPOT (ms)",
+    "p99_tpot_ms": "P99 TPOT (ms)",
+    "mean_latency_ms": "Mean Latency (ms)",
+    "std_latency_ms": "Std Latency (ms)",
+    "median_latency_ms": "Median Latency (ms)",
+    "p99_latency_ms": "P99 Latency (ms)",
     "input_throughput": "Input Tput (tok/s)",
     "output_throughput": "Output Tput (tok/s)",
```

Comment:
We changed these recently to `output` and `total`

Reply:
Get it --- let me make corresponding change

Reply:
Done